src/components/ebay-pagination/examples/1-basic/template.marko

```diff
@@ -0,0 +1,13 @@
+<ebay-pagination aria-label-prev="Previous page" aria-label-next="Next page" curr-text="Results Pagination - Page 1">
+    <ebay-pagination-item href="#" previous></ebay-pagination-item>
+    <ebay-pagination-item href="#">1</ebay-pagination-item>
+    <ebay-pagination-item href="#" current>2</ebay-pagination-item>
+    <ebay-pagination-item href="#hh">3</ebay-pagination-item>
+    <ebay-pagination-item href="#">4</ebay-pagination-item>
+    <ebay-pagination-item href="#">5</ebay-pagination-item>
+    <ebay-pagination-item href="#">6</ebay-pagination-item>
+    <ebay-pagination-item href="#">7</ebay-pagination-item>
+    <ebay-pagination-item href="#">8</ebay-pagination-item>
+    <ebay-pagination-item href="#">9</ebay-pagination-item>
+    <ebay-pagination-item href="#" next disabled></ebay-pagination-item>
```

Comment:
Just curious about the decision to make prev/next the same as `item`s, rather than using attributes like `prev-url` and `next-url` on the root.

Reply:
There might be other html-attributes like tracking that might need to be added so I made them an item. Otherwise the root would become too cluttered

Reply:
I can see it either way. But I can definitely see it being cluttered, especially as teams will be adding translated strings for all the accessibility attributes.

@yomed Do you feel using an attribute fits the style of what we're doing? More HTML-like?

Reply:
Ok, makes sense to keep it off the root if you anticipate more usage there. Did you also consider splitting these out into something like `<ebay-pagination-control>`? For me, it seems like the current pattern is a bit strange since the controls don't share a lot with the actual items, but I also haven't been exposed to all the use cases.